src/TUTORIAL.md

```diff
@@ -1016,7 +1016,26 @@ This is similar to what we've already done in the `accounts.js` handler, but mor
 
 ### Preventing Referred Records Deletion
 
-Another situation we must gracefully prevent is deleting _Product_ and _Account_ records when _Order_ records exist for them. Let's add the appropriate hooks to our `account.js` handler:
+Another situation we must gracefully prevent is deleting _Product_ and _Account_ records when _Order_ records exist for them. 
+
+Currently, (assuming a product with ID 1 has been purchased by a customer with an account which also has an ID of 1)... 
+
```

Comment:
Not sure about the ellipses. Stylistically, nowhere else in the tutorial we use it. I'd rather have the flow without those. If it's a continuation of the current sentence with a code example block, then next line simply starts with a lowercase word. If it's a new sentence that starts after a code example block, then it starts with an uppercase.